src/pg_cron.c

```diff
@@ -1540,10 +1539,10 @@ ManageCronTask(CronTask *task, TimestampTz currentTime)
 				break;
 			}
 
-			start_time = GetCurrentTimestamp();
+			task->lastStartTime = GetCurrentTimestamp();
```

Comment:
this should already be set when we change state to task->state = CRON_TASK_BGW_START;

seems a bit confusing to get another timestamp

removing this line and keeping the change below seems like a slight improvement

(probably lastStartTime was introduced after this line)